### ISSUE TITLE
get_Omega returns values above 1

### ISSUE THREAD

**Issue by Ni-Ar** [Pappulab/localCIDER#6]
Hi,

I tried to address this question to Alex but still haven't hear back from him. 

When I use the `get_Omega()`  function I noticed that some proteins return a value higher than 1. According to [the publication](https://www.pnas.org/content/pnas/110/33/13392.full.pdf) the Omega value should be between 0 and 1.
For example,  `get_Omega()` on the protein fasta sequence  below (Yeast Rnq1p) returns a value of ~ 1.371. Is this normal?
I'm running localcider version: 0.1.14 with Python 3.7.3

>Rnq1p
MDTDKLISEAESHFSQGNHAEAVAKLTSAAQSNPNDEQMSTIESLIQKIAGYVMDNRSGG
SDASQDRAAGGGSSFMNTLMADSKGSSQTQLGKLALLATVMTHSSNKGSSNRGFDVGTVM
SMLSGSGGGSQSMGASGLAALASQFFKSGNNSQGQGQGQGQGQGQGQGQGQGSFTALASL
ASSFMNSNNNNQQGQNQSSGGSSFGALASMASSFMHSNNNQNSNNSQQGYNQSYQNGNQN
SQGYNNQQYQGGNGGYQQQQGQSGGAFSSLASMAQSYLGGGQTQSNQQQYNQQGQNNQQQ
YQQQGQNYQHQQQGQQQQQGHSSSFSALASMASSYLGNNSNSNSSYGGQQQANEYGRPQQ
NGQQQSNEYGRPQYGGNQNSNGQHESFNFSGNFSQQNNNGNQNRY

**Comment by alexholehouse** [comment]
Hi Ni-Ar,

Thanks for your patience and sorry for slow response time :-) 

In principle these patterning parameters should always exist in the limits of 0 to 1. However, for sequences with a composition that is highly depleted in one or more of the types of residues being in a patterning group, a value greater than 1 can be obtained. From a biological standpoint this of limited practical concern. In general, for a sequence in which a patterning parameter yields a value greater than 1 is also well beyond the limit in which that parameter would have any real meaning due to the sequence composition.

So in this case, actually the E/D/R/K/P group used for calculating Omega has just 32 residues (out of 405 total - ~8%) so the GLOBAL patterning of these residues is probably. However (and I realize this is unhelpful) this does not mean that local clustering of residues isn't important, it's just these patterning parameters were developed in the limit of sequences were there are close to the same number of residues in each of the two groups. Actually to be honest 8% is not outrageous, we find that 10% or higher seems reasonable to assess patterning - my hunch is the 200 residue region depleted for charged/proline residues is a major contributor here. 

Mathematically, Omega is actually doing what it's meant to do - i.e. the Omega is asking "to what extent are the E/D/R/K/P residues distributed across the sequence compared to the average density (delta) normalized by the delta value obtained if all the E/R/D/K/P residues were grouped together (delta_max). The problem is that for a sufficiently long and E/R/D/K/P depleted sequences grouping all the residues together actually yields an Omega that is smaller than one that might be obtained if the residues were segregated another way.

Basically the conclusion you can draw from Omega in this case is that Rnq is highly E/R/K/D/P segregated!

**Comment by Ni-Ar** [comment]
Hi Alex,

thank you very much for the detailed  explanation! 
Initially I thought it was some sort of bug, but now I got how such high values might occur. 

Cheers!
